.gitignore

```diff
@@ -42,9 +26,9 @@ x64/
 ipch/
 *.aps
 *.ncb
-*.opensdf
-*.sdf
-*.cachefile
+
+=======
+>>>>>>> 96e5f232be906b4eda4d13ee9b2ecc1ba5ebbb6f
```

Comment:
Shouldn't be here. Looks like a _Merge Conflict_ Marker

Reply:
Fixed